docs/tutorial/volumes/rwx-with-nfs.md

```diff
@@ -0,0 +1,97 @@
+## Introduction
+
+Kubernetes support many types of volumes. A Pod can use any number of volume types simultaneously. One of the most useful types of volumes in Kubernetes is `NFS`.  A NFS volume can be accessed from multiple pods at the same time. This is really useful for running applications that need a filesystem that’s shared between multiple application servers.  NFS is a commonly used solution to provide ReadWriteMany(RWX) volumes on block storage in Kubernetes. This server offers a PersistentVolumeClaim (PVC) in RWX mode so that multiple applications can access the data in a shared fashion. In many cases, cloud block storage providers or OpenEBS volumes are used as persistent backend storage for these NFS servers to provide a scalable and manageable RWX shared storage solution. 
+
+OpenEBS Dynamic NFS PV provisioner can be used to dynamically provision NFS Volumes using different kinds of block storage available on the Kubernetes nodes. OpenEBS NFS provisioner is a kernel based server and thus it requires the NFS related packages has to be preinstalled on the required hosts. 
+
+In this document, we will explain how you can easily set up a NFS solution using OpenEBS block storage in your K8s cluster.  We will add a 100G disk to each node. These disks will be consumed by CSI based cStor pool and later we will create a cStor storage class and use this storage class as the backend storage for OpenEBS NFS provisioner. The deployment of OpenEBS NFS provisioner will create a storage class where provisioner will be OpenEBS NFS provisioner which uses cStor storage as the backend storage for provisioning persistent volume for NFS based applications. The recommended configuration is to have at least three nodes for provisioning at least 3 volume replicas for each volume and an unclaimed external disk to be attached per node to create cStor storage pool on each node with striped manner .  
+
+
+
+## Configuration workflow
+
+1. [Meet Prerequisites](/docs/tutorial/volumes/rwx-with-nfs.md#meet-prerequisites)
+
+2. [Installing OpenEBS NFS Provisioner](/docs/tutorial/volumes/rwx-with-nfs.md#installing-openebs-nfs-provisioner)
+
+
+### Meet Prerequisites
+
+- OpenEBS should be installed first on your Kubernetes cluster. The steps for OpenEBS installation can be found [here](https://docs.openebs.io/docs/next/installation.html). 
+- Perform cStor configuration and the steps can be found [here](https://github.com/openebs/cstor-operators/blob/master/docs/quick.md). cStor configuration includes installation of cStor operator, provisioning of a CStorPoolCluster(CSPC) and then creation of  a StorageClass that will consume the created CSPC pool. 
```

Comment:
should this be written as `Perform cStor configuration by following the steps mentioned [here]`